### ISSUE TITLE
Copy subtitles to clipboard when window is not active.

### ISSUE THREAD

**Issue by Surgetale** [asbplayer/asbplayer#478]
I am using a Texthooker in a second window (on second monitor) and when I look up words the subtitles won't get copied to clipboard anymore, until the window with the video player comes back in focus. Can this be fixed or added as a feature?

**Comment by killergerbah** [comment]
I don't think this is possible since this is likely enforced by the browser itself.

**Comment by luccamilfont** [comment]
My solution to this was to simply make an AHK script that auto focuses my window every second

```
#Persistent
SetTimer CheckWindow, 1000

CheckWindow:
	if WinExist("Anime")
    		WinActivate
Return

F2::Pause
Esc::ExitApp
```

Janky, but works :)
